contactsummary/js/test/controllers/key-dates.controller.spec.js

```diff
@@ -118,6 +114,36 @@ define([
             expect(controller.activeRoles).toBe(0);
           });
         });
+
+        describe('when end date of a job roles is today', function () {
```

Comment:
I think we need a test on "is yesterday"

Reply:
I think following will cover it, what do you say?
```js
describe('when end date of a job roles is less than today', function () {
   // ...
});
```